### DISCUSSION TITLE
register runner to organization

### DISCUSSION THREAD

**Discussion by igagis** [ChristopherHX/github-act-runner#3] [Q&A] [answered]
The README's instructions mention `<your github repository url>`, but I need to register the runner to an organization, so that is used for all the repos from that org. Is that possible?

**Comment by ChristopherHX** [comment, marked as answer]
I haven't thought about register the runner to an organization while writing my readme.
It seems the url is passed directly to github.com, so it should just work.

E.g.
```
github-actions-act-runner-windows-amd64.exe Configure --url https://github.com/Mathpp --token TTTTTTTTT2T2TTTTTTT2T2TT22T2T
```

Confirmed, it works for the default group
![image](https://user-images.githubusercontent.com/44845461/125696899-1debca46-5f71-4805-af4f-963544705774.png)

**Reply by igagis** [reply]
I'm still trying to configure it. After downloading about 3Gb of go packages, getting the following error:
```
build main: cannot find module for path embed
```
any idea how to fix it?

**Reply by ChristopherHX** [reply]
Please download it from https://github.com/ChristopherHX/github-actions-act-runner/releases/latest there are prebuilds for various systems (chmod +x) and replace `go run main.go` with `./github-actions-act-runner-GOOS-GOARCH`

The prebuilds don't need the go sdk.

> any idea how to fix it?

You need go 1.16 to get this working via `go run`, I think you have an older go executable.

**Reply by igagis** [reply]
Downloaded `linux-arm` variant. Registered successfully. Failed to run jobs.
For example see this run: https://github.com/cppfw/utki/runs/3071412644?check_suite_focus=true
it fails on `Set up job` step.
Runner output for 2 jobs:
```
Failed to upload timeline
Timeline Updated
{"path":"logs/06d3f99a-d3c7-40bc-bc1a-6fba85d180db","lineCount":4,"createdOn":"2021-07-14T22:20:58.66Z","lastChangedOn":"2021-07-14T22:20:58.8Z","id":110,"location":null}
Timeline Updated
Finishing Job
Finished Job
Failed to upload timeline
Timeline Updated
{"path":"logs/3198db01-6ddd-4d58-8c66-115c5d4fa1db","lineCount":4,"createdOn":"2021-07-14T22:21:03.12Z","lastChangedOn":"2021-07-14T22:21:03.257Z","id":112,"location":null}
Timeline Updated
Finishing Job
Finished Job
```

**Reply by igagis** [reply]
figured it out, I made a new user for running this runner and forgot to add it to `docker` group.

**Reply by igagis** [reply]
This runner definitely doesn't have the https://github.com/actions/runner/issues/1168 problem.
